docs/docs/dynamo-store/access-strategies.md

```diff
@@ -0,0 +1,73 @@
+# Access Strategies
+
+The access strategy allows you to opt into custom loading or reloading
+algorithms. For almost all categories in almost all most systems, `Unoptimized`
+is a good choice; you can always switch later.
+
+
+# Unoptimized
+
+The first, and likely most used access strategy is the unoptimized one. It'll
+not apply any optimised reading logic. When loading a stream for the first time
```

Comment:
optimised -> optimized!

Reply:
C'mon man! Expected you to represent the UK here 😂

Reply:
If you'd mentioned supporting the British Isles, perhaps :P its more the heading on L8 that prompts this!